test/integration/api.bats

```diff
@@ -48,3 +48,50 @@ function teardown() {
        [ "${#lines[@]}" -eq  2 ]
        [[ "${lines[1]}" == *"false"* ]]
 }
+
+
+@test "docker create/start/stop/restart/kill should return success" {
+	start_docker 1
```

Comment:
can you use more than one daemon, like 3
so we are sure that the calls go to the right node